lib/Sema/TypeCheckAvailability.h

```diff
@@ -117,7 +118,7 @@ class ExportContext {
   ExportContext(DeclContext *DC,
                 AvailabilityContext runningOSVersion,
                 FragileFunctionKind kind,
-                bool spi, bool exported, bool implicit, bool deprecated,
+                bool spi, bool exported, bool implicit, bool deprecated, bool IgnoresDeprecation,
                 Optional<PlatformKind> unavailablePlatformKind);
```

Comment:
Oh, I think this will also need a change to the constructor definition in TypeCheckAvailability.cpp. Have you been able to set up a build locally?

Reply:
Locally? Can’t really, my computer runs out of space, lemme edit the constructor